packages/api-gateway/src/api/attributesApiConverter.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  agreementApi,
+  tenantApi,
+  apiGatewayApi,
+} from "pagopa-interop-api-clients";
+import { isDefined, toSetToArray } from "pagopa-interop-commons";
+import {
+  verifiedAttributeToAttributeValidityState,
+  certifiedAttributeToAttributeValidityState,
+  declaredAttributeToAttributeValidityState,
+} from "./tenantApiConverter.js";
+
+export function toApiGatewayAgreementAttributes(
+  agreement: agreementApi.Agreement,
+  tenant: tenantApi.Tenant
+): apiGatewayApi.Attributes {
+  return {
+    verified: toSetToArray(
```

Comment:
Just curious, what does `toSetToArray` do in this case?

Reply:
It's to remove duplicates ( Array >>> Set (no duplicates) >>> Back to Array).
Probably it's better if I rename the function in `removeDuplicates` or similar :)

Reply:
But here this should not be working since the array passed in  is not made of primitive values but objects, and the `Set` does not remove duplicates by making a deep equal, but through a shallow one

Reply:
just to avoid code duplication, I remember the same function in one of Paolo's PRs

Reply:
It is the same function! It's from shared utils in the commons package.
However, I did not think about the deep vs shallow equality 🥲 
I'll need a better way to do it here...

Reply:
Done!